x-pack/plugins/security_solution/public/timelines/components/timeline/data_providers/provider_item_actions.tsx

```diff
@@ -138,7 +138,7 @@ export const getProviderActions = ({
       timelineType === TimelineType.template
         ? {
             className: CONVERT_TO_FIELD_CLASS_NAME,
-            disabled: isLoading,
+            disabled: isLoading || operator === IS_ONE_OF_OPERATOR,
```

Comment:
Thanks for disabling the option to covert the `IS_ONE_OF` operator to a template field.

Everything in the screenshot below is working as expecetd:

![working-as-expected](https://user-images.githubusercontent.com/4459398/193941852-8f166280-c7e8-43d9-aa5c-0a3912c95bc0.png)

The `Add field` case, shown below is also working as expected:

![add-field-timeline-template](https://user-images.githubusercontent.com/4459398/193942007-9174525c-3647-4d26-bf7c-94d64d07da37.png)

There are a two additional places in the UI where similar checks are necessary (to disable the option to convert the field to a template field):

1) The `is one of` and `is not one of` operators should not be shown in the `Add template field` scenario, shown in the screenshot below:

![add-template-field-timeline-template](https://user-images.githubusercontent.com/4459398/193942288-adc77d88-4cb8-4513-8e80-e30331344bae.png)

2) The `Convert to template field` option in the popover menu below should not be shown when the operator is `is one of`, or `is not one of`, per the screenshot below:

![popover-menu](https://user-images.githubusercontent.com/4459398/193942463-398fb931-2e01-4178-8ad8-17c16fcf14d5.png)

Reply:
fixed!